components/02-elements/swatch/_swatch.scss

```diff
@@ -1,31 +1,22 @@
-$swatch__font-size                    : $font-size-small !default;
-$swatch__margin                       : 15px !default;
-$swatch__width                        : 48px !default;
-$swatch__transition                   : $transition-base !default;
-
-$swatch__background                   : $white !default;
-$swatch__background--active           : $color-primary !default;
-
-$swatch__label-color                  : $gray-light !default;
-$swatch__label-color-hover            : $color-primary !default;
-$swatch__label-color--active          : $white !default;
-
-$swatch__border                       : 3px solid $gray-light !default;
-$swatch__border-color-hover           : $color-primary !default;
-$swatch__border-color--active         : $color-primary !default;
-
-// Variables for color swatches
-$swatch__outline--color               : 3px solid $white !default;
-$swatch__outline-color-hover--color   : $color-primary !default;
-$swatch__outline-color--color--active : $color-primary !default;
-
-$swatch__border-color--color          : $white !default;
-$swatch__border-color-hover--color    : $white !default;
-$swatch__border-color--color--active  : $white !default;
+$swatch__margin                            : $spacer !default;
+$swatch__transition                        : $transition-base !default;
+$swatch__title-font-size                   : $font-size-base !default;
+$swatch__option-sizing                     : 48px !default;
+$swatch__option-sizing-screen-m            : 32px !default;
+$swatch__option-margin                     : 0 $spacer--medium $spacer--medium 0 !default;
+$swatch__option-background                 : $white !default;
+$swatch__option-color                      : $gray-dark !default;
+$swatch__option-border                     : 1px solid $white !default;
+$swatch__option-border-color-hover         : $color-primary !default;
+$swatch__option-border-color--active       : $color-primary !default;
+$swatch__option-width-screen-m--attributes : 42px !default;
+$swatch__option-height-screen-m--attributes: 48px !default;
+$swatch__option-width-screen-l--attributes : 54px !default;
+$swatch__option-height-screen-l--attributes: 64px !default;
```

Comment:
in those variable we have 2 modifiers, first it's media query and we should use `small`, `medium`, `large` etc, IMO it should be like:  `$swatch__option-width--medium--attributes` etc

Reply:
ok